src/js/services.js

```diff
@@ -45,4 +45,15 @@ const getFilterTimeline = (filter) => {
 	return fetch('http://localhost:8080/api/1.0/twitter/tweet/filter?filter=' + filter)
 }
 
-export{getHomeTimeline, getUserTimeline, getFilterTimeline, renderedTimeline};
\ No newline at end of file
+const postTweet = (tweet) => {
+	let data = {"name": "mojo", "message": tweet};
```

Comment:
why is name 'mojo'?

Reply:
it was just defualt name, ill change it to "default name"

Reply:
nvm I just didnt pass anything for the name attribute